### DISCUSSION TITLE
Span contenteditable doesnt work inside a custom node

### DISCUSSION THREAD

**Discussion by navaniharsh31** [xyflow/xyflow#2918] [Q&A] [answered]
Using a library (`@yaireo/tagify`) which uses this, didnt seem to be working. Tried adding a span with contentEditable manually, it still didnt work inside a custom node. Any workaround?

**Comment by navaniharsh31** [comment, marked as answer]
Adding "nodrag" did the job

**Reply by MatheusRoichman** [reply]
What if I want to be able to drag the element?

**Reply by MetaMmodern** [reply]
For anyone else confused on what that means: add `className="nodrag"` to the editable element. 
Docs: https://reactflow.dev/learn/customization/custom-nodes#nodrag

**Comment by Stettzy** [comment]
We owe you a beer sir. Saved us going bald.

**Comment by ulysse0** [comment]
I can confirm that we owe you more than a beer, sir.

**Comment by zaidnadaff** [comment]
You are the besttt!!!!!
